builtin/logical/nomad/backend.go

```diff
@@ -0,0 +1,37 @@
+package nomad
+
+import (
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func Factory(conf *logical.BackendConfig) (logical.Backend, error) {
+	b := Backend()
+	if err := b.Setup(conf); err != nil {
+		return nil, err
+	}
+	return b, nil
+}
+
+func Backend() *backend {
+	var b backend
+	b.Backend = &framework.Backend{
+		Paths: []*framework.Path{
+			pathConfigAccess(),
```

Comment:
Please add a `config/lease` path for configuring the `ttl` and `max_ttl` for the backend.

Reply:
Fixed in https://github.com/hashicorp/vault/pull/3401/commits/d1e3eff61808f74f97b899dd6c161b83d4fab6e9